packages/loot-core/migrations/1704572023730_add_account_sync_source.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN TRANSACTION;
```

Comment:
We might also want to add a migration script to populate this column with `gocardless` so that we don't need to add code to default to gocardless when this column is empty.

Reply:
Is there a precedent for doing this? Are all migrations guaranteed to run first?

Reply:
I see that we are defaulting to GoCardless when `account_sync_source` is empty: https://github.com/actualbudget/actual/pull/2188/files#diff-c9b9c2688ecbd5ed8814e71d09f631d2bece3b987ba9b4a04e06cfe07e1b27f2R1387

I believe this is done to handle synced GoCardless accounts prior to this PR? If that's the case why don't we just populate the new `account_sync_source` column with `gocardless` if an exsting account's `account_id` has a value set? Then we can remove those else blocks that default to gocardless.